README.md

```diff
@@ -1 +1,34 @@
-# Operator Marketplace
+# Marketplace Operator
+
+## Project Status: pre-alpha
+The project is currently pre-alpha and it is expected that breaking changes to the API will be made in the upcoming releases.
+
+## Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Have an OpenShift cluster with Operator Lifecycle Manager [installed](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md).
+2. Be logged in as a user with the RBAC permissions required to deploy this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
```

Comment:
How about the following?
1. Be logged in as a user with Cluster Admin role.
   1. This is a stop gap measure until the RBAC permissions are defined

Reply:
Done.